src/webui/www/private/scripts/client.js

```diff
@@ -579,18 +582,31 @@ window.addEvent('load', function() {
                         // re-select previously selected rows
                         torrentsTable.reselectRows(torrentsTableSelectedRows);
                 }
-                clearTimeout(syncMainDataTimer);
-                syncMainDataTimer = syncMainData.delay(getSyncMainDataInterval());
+                requestInProgress = false;
+                syncData(getSyncMainDataInterval())
+                $('syncing_div').set('text', '');
             }
-        }).send();
+        });
+        requestInProgress = true;
+        request.send();
     };
 
     updateMainData = function() {
         torrentsTable.updateTable();
-        clearTimeout(syncMainDataTimer);
-        syncMainDataTimer = syncMainData.delay(100);
+        syncData(100);
     };
 
+    syncData = function(delay)
```

Comment:
shouldn't it be `const syncData = function(delay)`?

Reply:
Ok, done.